src/test/resources/employee_curd_operation.feature

```diff
@@ -0,0 +1,29 @@
+Feature: employee_curd_operation
+	Employee CRUD operation
+	
+	Scenario: Save employee record
+		Given passing valid employee record
+		When  employee should successfully saved to db
+		Then  it should return saved employee object
+
+	Scenario: is to Get employee by id
```

Comment:
Scenario: Get employee by id.. change others accordingly

Reply:
resolved as part of below commit 

https://github.com/LokeshReddyExamples/springBootAssignment/pull/4/commits/8a6230e2a67b3dd419b3c263d7c7aa9a9e44c5d6